build.ps1

```diff
@@ -83,6 +86,7 @@ param(
   [string] $SwiftDebugFormat = "dwarf",
   [string[]] $SDKs = @("X64","X86","Arm64"),
   [string] $ProductVersion = "0.0.0",
+  [string] $PinnedToolchain = "swift-DEVELOPMENT-SNAPSHOT-2023-08-12-a",
```

Comment:
maybe `PreviousToolchain`? It describes a bit better why we need it.

Reply:
It isn't a previous toolchain, its a pinned toolchain that we use to build parts of it.

Reply:
In some cases this may even be a future toolchain (like on 5.9).